air/src/air/context.rs

```diff
@@ -250,13 +250,9 @@ impl<B: StarkField, P> AirContext<B, P> {
         self.aux_transition_constraint_degrees.len()
     }
 
-    /// Returns the index of the auxiliary column which implements the Lagrange kernel, if any
+    /// Returns the index of the auxiliary column which implements the Lagrange kernel, if any.
     pub fn lagrange_kernel_aux_column_idx(&self) -> Option<usize> {
```

Comment:
nit: I would rename this into just `lagrange_kernel_column_idx()`. I think the "auxiliary" part is kind of implied - but also, feel free to ignore.

Reply:
Makes sense, changed